examples/elf/tests/helloxx/.gitignore

```diff
@@ -2,4 +2,4 @@ hello++1
 hello++2
 hello++3
 hello++4
-hello++5
\ No newline at end of file
+hello++5
```

Comment:
It is really impossible to see.  Is there a newline after hello++5 or not?  Where there two before and now only one?  Or is there no newline now.  That would be bad.

Reply:
No, you can see there is a red dot at the end of line before change which mean no newline there:
![image](https://user-images.githubusercontent.com/18066964/74040901-998f1280-49ff-11ea-939a-929b2f7c99a0.png)